rust/src/lightning.rs

```diff
@@ -152,28 +152,15 @@ pub async fn open_channel(
     channel_amount_sat: u64,
     initial_send_amount_sats: u64,
 ) -> Result<()> {
-    let config = UserConfig {
-        channel_handshake_limits: ChannelHandshakeLimits {
-            // lnd's max to_self_delay is 2016, so we want to be compatible.
-            their_to_self_delay: 2016,
```

Comment:
🤔 Was this line removed on purpose? I feel it is unrelated to the changes described.

Reply:
Not really, but not needed for our use at the moment.

Reply:
We will notice once we try 😂